test/UnitTest/HelperTests/InstanceEventHelperTest.cs

```diff
@@ -43,13 +43,13 @@ public void RemoveDuplicateEvents_GivenListWithOneEvent_ReturnsListWithSameEvent
         }
 
         [Fact]
-        public void RemoveDuplicateEvents_GivenListWithTwoEqualEvents_ReturnsListWithSingleEvent()
+        public void RemoveDuplicateEvents_GivenListWithTwoEqualSavedEvents_ReturnsListWithSingleEvent()
         {
             // Arrange
             List<InstanceEvent> instanceEvents = new List<InstanceEvent>
             {
-                new InstanceEvent(),
-                new InstanceEvent()
+                new InstanceEvent() { EventType = InstanceEventType.Saved.ToString() },
+                new InstanceEvent() { EventType = InstanceEventType.Saved.ToString() }
```

Comment:
Could you add one entry of another type and change the assert to test for 2 entries?

Reply:
Done